src/test/test_ep_cred_common.h

```diff
@@ -46,7 +46,8 @@
 
 #define OPC_MY_PROTO    (0x01000000)
 #define OPC_PING	(0x01000000)
-#define OPC_SHUTDOWN    (0x01000001)
+#define OPC_PING_FRONT	(0x01000001)
+#define OPC_SHUTDOWN    (0x01000002)
```

Comment:
Doers changing an opcode value create version mismatch issues? Should PING_FRONT be an add on to the next available opcode? Just wondering if versioning would be an issue here.

Reply:
If this was production client/server situation then yes, you would be right. However given that its a test code and we ship client+server together, it should not be an issue.

Reply:
@vschhabra is correct here though, we should really bump the protocol version number here because this is a protocol change.  In practice we probably only need to do this for versions/builds which have been deployed outside of the group or test systems though.

Reply:
for a production version i would agree, for test code at this stage in my view its not worth the hassle, especially when we will be doing test rewrites with pmix removal.